plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/utils/qemu/QemuImg.java

```diff
@@ -234,14 +235,19 @@ public void create(final QemuImgFile file, final Map<String, String> options) th
      * @return void
      */
     public void convert(final QemuImgFile srcFile, final QemuImgFile destFile,
-                        final Map<String, String> options, final String snapshotName) throws QemuImgException {
+                        final Map<String, String> options, final String snapshotName) throws QemuImgException, LibvirtException {
         Script script = new Script(_qemuImgPath, timeout);
         if (StringUtils.isNotBlank(snapshotName)) {
             String qemuPath = Script.runSimpleBashScript(getQemuImgPathScript);
             script = new Script(qemuPath, timeout);
         }
 
         script.add("convert");
+        Long version  = LibvirtConnection.getConnection().getVersion();
+        if (version > 2009000) {
```

Comment:
@GutoVeronezi 
I checked qemu codebase, this is introduced in 2.10.0

so I suggest to use 
```
if (version >= 2010000) {
```

qemu 2.9.1 does not support it.